katas/content/linear_algebra/index.md

```diff
@@ -224,11 +224,11 @@ Another, equivalent definition highlights what makes this an interesting propert
 $$A^{-1}(AB) = A(A^{-1}B) = B$$
 $$(CA)A^{-1} = (CA^{-1})A = C$$
 
-A square matrix has a property called the **determinant**, with the determinant of matrix $A$ being written as $|A|$. A matrix is invertible if and only if its determinant isn't equal to $0$.
+A square matrix has a property called the **determinant**, with the determinant of matrix $A$ being written as $|A|$. A matrix is invertible if and only if, its determinant isn't equal to $0$.
```

Comment:
The comma seems out of place. It read better before the change.

Reply:
```suggestion
A square matrix has a property called the **determinant**, with the determinant of matrix $A$ being written as $|A|$. A matrix is invertible if, and only if, its determinant isn't equal to $0$.
```